src/vos/evtree.c

```diff
@@ -2389,7 +2393,7 @@ typedef int (cmp_rect_cb)(struct evt_context *tcx, const struct evt_rect *mbr,
 static int
 evt_common_insert(struct evt_context *tcx, struct evt_node *nd,
 		  umem_off_t in_off, const struct evt_entry_in *ent,
-		  bool *changed, cmp_rect_cb cb)
+		  bool *changed, cmp_rect_cb cb, bool tx_added)
```

Comment:
Is this parameter necessary? I don't think we need it, if the tn_nr is changed in this function, just simple call add_range() to tn_nr is fine. PMDK maintains a snapshot tree internally, I think it's ok to add same range multiple times in same transaction.